api_tests/files/serializers/test_file_serializer.py

```diff
@@ -65,12 +76,73 @@ def test_file_serializer(self, file_one):
         # check download file link with path
         assert download_base.format(path) in data['links']['download']
 
+        # check html file link with path
+        assert download_base.format(path) in data['links']['html']
+        assert mfr_url in data['links']['html']
+
         # check download file link with guid
         guid = file_one.get_guid(create=True)._id
         req = make_drf_request_with_version()
         data = FileSerializer(file_one, context={'request': req}).data['data']
         assert download_base.format(guid) in data['links']['download']
 
+        # check html file link with guid
+        assert download_base.format(guid) in data['links']['html']
+        assert mfr_url in data['links']['html']
+
+        # check download/html link for folder
+        folder = node.get_addon('osfstorage').get_root().append_folder('Test_folder')
+        folder.save()
+        req = make_drf_request_with_version(version='2.2')
+        data = FileSerializer(folder, context={'request': req}).data['data']
+        assert 'html' not in data['links']
+        assert 'download' not in data['links']
+
+    def test_serialize_preprint_file(self, preprint, primary_file):
+        req = make_drf_request_with_version(version='2.2')
+        data = FileSerializer(primary_file, context={'request': req}).data['data']
+        download_link = data['links']['download']
+        html_link = data['links']['html']
+        mfr_url = get_mfr_url(preprint, 'osfstorage')
+
+        assert download_link in html_link
+        # Check html file link with path
+        assert html_link == mfr_url + '/render?url=' + download_link + '?direct%26mode=render'
```

Comment:
This assert and the ones below it would be a bit clearer with `'{}'.format()`

Reply:
you're right!